internal/security/proxy/requestor.go

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * @author: Tingyu Zeng, Dell
+ *******************************************************************************/
+package proxy
+
+import (
+	"crypto/tls"
+	"net/http"
+	"time"
+)
+
+type Requestor interface {
```

Comment:
this interface is already effectively defined here: https://github.com/edgexfoundry/edgex-go/blob/master/internal/interface.go#L5 
why not reuse that one?

Reply:
Resolved https://github.com/edgexfoundry/edgex-go/pull/1660